pages/common/fio.md

```diff
@@ -0,0 +1,18 @@
+# fio
+
+> Flexible I/O tester
+> Tool that will spawn a number of threads or processes doing a particular type of I/O action.
+> More examples: <https://docs.oracle.com/en-us/iaas/Content/Block/References/samplefiocommandslinux.htm>.
```

Comment:
We only use 1 link per page
```suggestion
```

Reply:
It's a pity that link to the best examples has to be removed :/